libopencm3.nix

```diff
@@ -25,8 +25,15 @@ stdenvNoCC.mkDerivation rec {
     python311
     gcc-arm-embedded-13 # arm-gnu-toolchain-13.2.rel1
   ];
+  devicesPatch = writeText "devices-patch.txt" ''
+    # Device otherwise missing from OpenCM3
+    stm32l4r5zi stm32l4 ROM=2048K RAM=256K RAM3=384K
+  '';
```

Comment:
```suggestion
```
I don't think you need that, because we feed in the devices.data separately anyway?

Reply:
done in https://github.com/pq-code-package/mlkem-c-embedded/commit/ee07cad0dd81afa57ed849349e73ac60da0d3f5d